src/components/Table/Budget/BudgetLineItemTable/BudgetLineItemTableView.tsx

```diff
@@ -143,7 +143,7 @@ function BudgetLineItemTableView({
 	financialYearOrgHash: { [key: string]: string };
 	currency: string;
 }) {
-	tableHeadings[3].label = "Amount " + `(${currency})`;
+	tableHeadings[3].label = `Amount (${currency})`;
```

Comment:
what is `3`?

Reply:
We have kept all of the table headings in seprate file so it can be used both in testing and component